### PR TITLE
1.2.3 release

### PR DESCRIPTION
**Pull request by programmatix**
In addition:
* Fixing DOC-9630.
* As per DOC-9630, N1QL is now SQL++.
* Removing a duplicate entry for the 1.2.2 release.

### CODE REVIEW

**Comment by maria-robobug** [comment]
Hmm, I'm seeing this error when building the code samples, any idea @programmatix?
> Failed to collect dependencies at com.couchbase.client:couchbase-transactions:jar:1.2.3: Failed to read artifact descriptor for com.couchbase.client:couchbase-transactions:jar:1.2.3: Could not transfer artifact com.couchbase.client:couchbase-transactions:pom:1.2.3 from/to maven-default-http-blocker (http://0.0.0.0/): Blocked mirror for repositories: [couchbase (http://files.couchbase.com/maven2, default, releases+snapshots)] -> [Help 1]

**Comment by maria-robobug** [comment]
Ah, not published on maven yet maybe? https://search.maven.org/artifact/com.couchbase.client/couchbase-transactions

File: modules/howtos/examples/TransactionsExample.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ static void create() {
         try {
             transactions.run((ctx) -> {
                 // 'ctx' is an AttemptContext, which permits getting, inserting,
-                // removing and replacing documents, performing N1QL queries, and committing or
+                // removing and replacing documents, performing SQL++ queries, and committing or
```
Comment (maria-robobug) [inline modules/howtos/examples/TransactionsExample.java]: We haven't yet started changing references of `N1QL` to `SQL++`, so I will revert this for now I'm afraid.

**Comment by programmatix** [comment]
Yes sometimes sonatype takes a little time to finish the release.

**Comment by programmatix** [comment]
Are we not doing SQL++ yet?  I was advised under DOC-9630 that we are.

**Comment by maria-robobug** [comment]
> Are we not doing SQL++ yet? I was advised under DOC-9630 that we are.

@programmatix we are currently discussing within the DOCS team to establish where/when we need to make these changes (more team wide concern than just SDK), so we're keeping these as is for now to keep things consistent. Or else it will look strange if some places still say N1QL and others SQL++.

**Review by maria-robobug — APPROVED** [review]

**Pull request merged**